File: Java/682.java
import java.util.Scanner;

/**
 *
 * 682 - Metidos en un Cajón - Iván - https://github.com/wildfireOfMine
 */
public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        final int casos = sc.nextInt();
        for (int i = 1; i <= casos; i++) {
            int cosa = sc.nextInt();
            System.out.println(cosa + " " + cosa * 2);
        }
    }

}

File: C++/682.cpp
#include <iostream>

using namespace std;

// 682 - Metidos en un Cajón - Iván - https://github.com/wildfireOfMine

int main()
{
    int loop = 0;
    cin >> loop;
    for (int i = 0; i<loop; i++) {
        int peak = 0;
        int legs = 0;
        cin >> peak;
        legs = peak*2;
        cout << peak << " " << legs<< endl;

    }
}
